signify-ts-test/test/utils/resolve-env.ts

```diff
@@ -196,6 +203,193 @@ export function resolveEnvironment(
           process.env.VLEI_VERIFIER || "NordLEI demo verifier not set",
         roleName: process.env.ROLE_NAME || "unicredit-datasubmitter",
         secrets: providedSecrets.split(","),
+        lei: process.env.LEI || "875500ELOZEL05BVXV37",
+      };
+      break;
+    default:
+      throw new Error(`Unknown test environment preset '${preset}'`);
+  }
+  console.log("Test environment preset: ", JSON.stringify(env));
+  return env;
+}
+
+export function resolveEnvironmentMultipleAid(
```

Comment:
I'd rather see the different environment maintained in resolveEnvironment, instead of duplicated. So either SIGNIFY_SECRETS should be upgraded to be what SIGNIFY_SECRETES_MULTI_AID has become, otherwise shouldn't this call resolveEnvironment by parsing the SIGNIFY_SECRETS_MULTI_AID and setting the SIGNIFY_SECRETS env to the first set, run everything given the resolveEnv with that set, then repeat again for the other sets. Otherwise won't we have a mess of environments to maintain?